### PR TITLE
[java-gen] Support loading CRDs from folder

### PR DESCRIPTION
**Pull request by andreaTP**
## Description

This implement loading CRDs from a folder. ( cc. @manusa )

Note:
I should better automate the automatic formatting ... sorry for the noise ...

## Type of change
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [X] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [X] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [ ] I Added [CHANGELOG](../CHANGELOG.md) entry regarding this change
 - [X] I have implemented unit tests to cover my changes
 - [ ] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

**Comment by andreaTP** [comment]
@manusa @oscerd @rohanKanojia 
this should be ready for review.

**Review by oscerd — APPROVED** [review]

**Review by rohanKanojia — APPROVED** [review]

File: java-generator/maven-plugin/src/main/java/io/fabric8/java/generator/maven/plugin/JavaGeneratorMojo.java [hunk 1]
```diff
@@ -30,9 +30,8 @@
     @Parameter(defaultValue = "${project}")
     private MavenProject project;
 
-    // TODO: Modify CRGeneratorRunner to allow passing in a directory too
     @Parameter(property = "fabric8.java-generator.source", required = true)
-    File crdFile;
+    File crdSource;
```
Comment (manusa) [inline java-generator/maven-plugin/src/main/java/io/fabric8/java/generator/maven/plugin/JavaGeneratorMojo.java:34]: maybe we should call this just `source`

And same for `targetDirectory` ---> `target`

Comment (andreaTP) [reply java-generator/maven-plugin/src/main/java/io/fabric8/java/generator/maven/plugin/JavaGeneratorMojo.java:34]: make sense, let me update the PR.

Comment (andreaTP) [reply java-generator/maven-plugin/src/main/java/io/fabric8/java/generator/maven/plugin/JavaGeneratorMojo.java]: done

**Review by manusa — APPROVED** [review]

**Pull request merged**